Add ?warn command (#6)
codewars/discord-bot: text/warn/promote.md

> You are receiving this message because you have been found promoting products and/or services outside of Codewars and Qualified.
> 
> Please take a moment to be reminded of the following rule from this server's #readme channel:
> 
> - No spam or self-promotion (server invites, advertisements, etc) without permission from @mods. This includes DMing fellow members.
> 
> Consider this a formal warning. If you insist on engaging in such behavior, we reserve the right to kick or ban you from the server indefinitely, and your Codewars account (if any) may be restricted.

Reviewer: ```suggestion
You've violated the following rule:

- No spam or self-promotion (server invites, advertisements, etc) without permission from @mods. This includes DMing fellow members.

If you insist on engaging in such behavior, we reserve the right to kick or ban you from the server indefinitely, and your Codewars account (if any) may be restricted. You've been warned.
```

Let's change this to `spam`.
